torch/linalg/__init__.py

```diff
@@ -309,26 +347,25 @@
 eigvals = _add_docstr(_linalg.linalg_eigvals, r"""
 linalg.eigvals(input, *, out=None) -> Tensor
 
-Computes the eigenvalues of a square
-matrix :attr:`input`, or of each such matrix in a batched :attr:`input`.
-
-The eigenvalues are not necessarily sorted.
-For real-valued input the resulting eigenvalues will always be of complex type.
+Computes the eigenvalues of a square matrix or a batch of matrices :attr:`input`.
 
-Supports input of float, double, cfloat and cdouble dtypes.
+Supports :attr:`input` of float, double, cfloat and cdouble dtypes.
+The eigenvalues will always be complex-valued, even if :attr:`input` is real.
 
-.. note:: When given inputs on a CUDA device, this function synchronizes that device with the CPU.
+.. note:: When :attr:`input` is on a CUDA device, this function synchronizes that device with the CPU.
 
-.. note:: The eigenvalues are computed using LAPACK's and MAGMA's `geev` routine.
+.. note:: This function is computed using LAPACK's and MAGMA's `geev` routine for
+          CPU and CUDA inputs, respectively.
 
-.. note:: See :func:`torch.linalg.eig` for a related function that computes both eigenvalues and eigenvectors.
+.. note:: This function doesn't support backpropagation. Use :func:`torch.linalg.eig`
+          instead, which also computes the eigenvectors.
```

Comment:
Is linalg.eig actually differentiable now?

Reply:
I'll implement that this week.